string obj
---
geometry_msgs/PoseStamped location
